cmd/nerdctl/run_cgroup_linux_test.go

```diff
@@ -283,6 +283,50 @@ func TestRunCgroupConf(t *testing.T) {
 		"cat", "memory.high").AssertOutExactly("33554432\n")
 }
 
+func TestRunCgroupParent(t *testing.T) {
+	t.Parallel()
+	base := testutil.NewBase(t)
+	info := base.Info()
+	containerName := testutil.Identifier(t)
+	defer base.Cmd("rm", "-f", containerName).Run()
+
+	switch info.CgroupDriver {
+	case "none", "":
+		t.Skip("test requires cgroup driver")
+	}
+
+	parent := "/foobarbaz"
+	if info.CgroupDriver == "systemd" {
+		// Path separators aren't allowed in systemd path. runc
+		// explicitly checks for this.
+		// https://github.com/opencontainers/runc/blob/016a0d29d1750180b2a619fc70d6fe0d80111be0/libcontainer/cgroups/systemd/common.go#L65-L68
+		parent = "foobarbaz.slice"
+	}
+
+	// cgroup2 without host cgroup ns will just output 0::/ which doesn't help much to verify
+	// we got our expected path. This approach should work for both cgroup1 and 2, there will
+	// just be many more entries for cgroup1 as there'll be an entry per controller.
+	cmd := base.Cmd(
+		"run",
+		"-d",
+		"--name",
+		containerName,
+		"--cgroupns=host",
+		"--cgroup-parent", parent,
+		testutil.AlpineImage,
+		"sleep",
+		"infinity",
+	)
+	cmd.Run()
+
+	id := cmd.Base.InspectContainer(containerName).ID
```

Comment:
base.InspectContainer?

Reply:
Yep, no need to chain off cmd, will change

Reply:
Fixed